pkg/intoto/intoto_test.go

```diff
@@ -1,45 +0,0 @@
-package intoto
```

Comment:
This is rather simple, but I hoped that at some point we would add more tests specific to CNAB here.

Reply:
you're right, there's no good reason to delete